keyboards/handwired/splittest/proton_c/readme.md

```diff
@@ -0,0 +1,11 @@
+# Proton C splittest
+
+To trigger keypress, short together pins *B14* and *B9* (marked on the PCB as *B3* and *B6*).
+
+## Wiring
+- Add switches to both Proton C across B3 and B6 pins
+- Add pull-up resistor to left side between VCC and F6
+- Add pull-down resistors to right side between GND and F6
+- Connect the following pins on both sides together: D0, D1, GND, VCC
```

Comment:
this is a bit ambiguous, I'm guessing you mean connect the pins on each half straight through?

i.e. A - D0 <---> D0 - B